mlflow/Dockerfile

```diff
@@ -9,11 +9,11 @@ RUN apt-get update && \
     apt-get clean
 
 RUN pip install google-cloud-storage
-RUN pip install psycopg2-binary==2.8.6
+RUN pip install psycopg2-binary==2.9.10
 
-ARG BOTO3_VERSION=1.7.12
-ARG MLFLOW_VERSION=1.3.0
-ARG SQLALCHEMY_VERSION=1.4.46
+ARG BOTO3_VERSION=1.35.39
+ARG MLFLOW_VERSION=1.26.1
+ARG SQLALCHEMY_VERSION=1.4.54
```

Comment:
Are these version bumps all related to new version of mlflow? Or we're just bumping them to use a newer version?

Reply:
I'm just bumping them to use a later version for now (the versions originally specified will block later versions of mlflow past version 1.26.1 to be installed). SQL alchemy's intentionally kept below version 2 because version 1.26.1 of Mlflow is only compatible with version 1 of it.